generators/app/templates/app/config/route-config.js

```diff
@@ -66,17 +72,38 @@ function getMethod(routeItem) {
     throw 'Undefined or empty "method" property in "lib/config/route.config.json"';
   }
 
-  var method = routeItem.method.toLowerCase();
-
-  switch(method) {
-    case 'get':
-    case 'put':
-    case 'post':
-    case 'delete':
-      return method;
-      break;
-    default:
-      throw 'Invalid REST "method" property in "lib/config/route.config.json": ' + method;
+  var method;
```

Comment:
This could be cleaner, there is code duplication here that could easily be refactored to move the code in the switch statement into another function.